### PR TITLE
WD-14408 - add assets on /solutions/ubuntu-os

### PR DESCRIPTION
**Pull request by lizzochek**
## Done

Added assets to the "Everywhere for everyone" section as per figma

## QA

- [demo link](https://canonical-com-1372.demos.haus/solutions/ubuntu-os)
- [figma](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=510-4340&node-type=frame&t=rBIwmAK7DpxxFpdm-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-14408](https://warthogs.atlassian.net/browse/WD-14408)

[WD-14408]: https://warthogs.atlassian.net/browse/WD-14408?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mattea-turic** [comment]
only comment I have is that the images here aren't the same height

Screenshot 2024-09-17 at 09 21 39
Screenshot 2024-09-17 at 09 22 43


Lmk if I can help if it's an assets thing, but I'm using the comp just straight from the figma library so they should be correct aspect ratio

**Comment by petesfrench** [comment]
@mattea-turic wouldn't this be a good candidate for the new [equal-row-height--wrap variant](https://vanillaframework.io/docs/patterns/equal-height-row#wrap-variant)?

File: templates/solutions/ubuntu-os/index.html [hunk 1]
```diff
@@ -248,8 +248,11 @@ <h2>Everywhere and for everyone</h2>
     </div>
     <div class="row p-section--shallow">
       <div class="col-9 col-start-large-4">
-        <div class="p-equal-height-row has-divider-1 has-divider-3">
+        <div class="p-equal-height-row has-divider-3">
```
Comment (petesfrench) [inline templates/solutions/ubuntu-os/index.html:251]:
```suggestion
        <div class="p-equal-height-row--wrap has-divider-3">
```
based on what mattea says, could make this a `--wrap` element

**Review by petesfrench — APPROVED** [review]
LGTM

**Pull request merged**